rpc_util.go

```diff
@@ -258,7 +258,8 @@ func (o PeerCallOption) after(c *callInfo, attempt *csAttempt) {
 }
 
 // WaitForReady configures the action to take when an RPC is attempted on broken
-// connections or unreachable servers. If waitForReady is false, the RPC will fail
+// connections or unreachable servers. If waitForReady is false and the
+// connection is in the TRANSIENT_FAILURE or SHUTDOWN states, the RPC will fail
```

Comment:
Technically we fail _all_ RPCs instantly if we are in `SHUTDOWN`.  Maybe just mention `TRANSIENT_FAILURE` here?

Reply:
Great point, thanks. I have made that edit. FWIW: I mostly copied this from https://github.com/grpc/grpc/blob/master/doc/wait-for-ready.md , which mentions both states explicitly.

Reply:
> FWIW: I mostly copied this from

My concern here was the "otherwise" leading off the following sentence.  It implied that SHUTDOWN+WFR would block instead of failing immediately.

Reply:
You are right! It does imply that. Thanks for the clarification: this will hopefully help my writing in the future :)